grpc.go

```diff
@@ -1273,6 +1273,9 @@ func (a *agentGRPC) RemoveContainer(ctx context.Context, req *pb.RemoveContainer
 				}
 			}
 		}
+		configJsonDir := filepath.Join("/run/libcontainer/", req.ContainerId)
```

Comment:
Hi  @bpradipt 
Here you just do the cleanup in case of the timeout == 0,  how about 
doing the cleanup at the  bottom of this function or put it a delay function?

Reply:
Hi @lifupan yeah sure. I'll make the changes and test it out.  Can you tell me what's the `timeout` is for? I didn't hit the else path during my local testing.